docs/source/running-autometa.rst

```diff
@@ -2,108 +2,271 @@
+    params.metagenome = "<path/to/assembly_dir/*.fasta>" 
+    params.interim = "<path/to/interim_results/>" 
+    params.processed = "<path/to/processed_results/>"
+
+.. note::
+    Wildcard characters will only be interpreted when "double quotes" are used
+
+Database directory
+^^^^^^^^^^^^^^^^^^
+
+Autometa uses the following NCBI databses throughout its pipeline:
+
+- Non-redundant `nr database <ftp://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/nr.gz>`_
+- `prot.accession2taxid.gz <ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz>`_
+- *nodes.dmp*, *names.dmp* and *merged.dmp* from `taxdump tarball <ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz>`_ 
+
+If you are running autometa for the first time you'll have to download these databases. Use ``autometa-update-databases --update-ncbi``. This will download the databases to the default path. You can check the default paths using ``autometa-config --print``. If you need to change the default download directory you can use ``autometa-config --section databases --option ncbi --value <path/to/new/ncbi_database_directory>``. See ``autometa-update-databases -h`` and ``autometa-config-h`` for full list of options.
+
+In your ``parameters.config`` file you also need to specify the directory where the different databases are present. Make sure that the directory path contains the following databases:
+
+- Diamond formatted nr file => nr.dmnd
+- Extracted files from tarball taxdump.tar.gz
+- prot.accession2taxid.gz
+
+.. code-block:: bash
+
+    // Find this section of code in parameters.config
+    // Update this path to folder with all NCBI databases
+    params.ncbi_database = "/Autometa/autometa/databases/ncbi"
+
+Runtime parameteres
+^^^^^^^^^^^^^^^^^^^
+
+You can configure the number of CPUs that each job should use.
+
+.. code-block:: bash
+
+    // Find this section of code in parameters.config
+    // Change the number of CPUs you want each job to use
+    params.cpus = 2
+
+Additional autometa parameteres
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can also adjust other pipeline parameteres that ultimate control how the binning would be done.
+
+*params.length_cutoff* : Smallest contiog you want binned (default is 3000bp)
+
+*params.kmer_size* : kmer size to use
+
+*params.kmer_norm_method* : Which normaization method to use. Choices are centered log-ratio transformation (clr), Autometa's clr implementation (am_clr) and isometric log-ratio transformation (default is am_clr)
+
+*params.kmer_pca_dimensions* : Number of dimensions to reduce the initial 512 dimension big k-mer matrix to (default is 50)
+
+*params.kmer_embed_method* :  kmer-embedding method. Choices are "sksne", "bhsne", "umap" (default is bhsne)
+
+*params.kmer_embed_dimensions* : Final dimensions you want to reduce the pca reduced kmer matrix to (default is 2).
+
+*params.kingdom* : Bins contigs belonging to which kingdom. Choices are "bacteria" and "archaea" (default is bacteria)
```

Comment:
```suggestion
*params.kingdom* : Bin contigs belonging to this kingdom. Choices are "bacteria" and "archaea" (default is bacteria)
```

We could by default go through marker annotation and binning for all kingdoms. This way, this parameter would be removed.